timesketch/api/v1/resources.py

```diff
@@ -1795,15 +1816,30 @@ def post(self, sketch_id):
                     'No timeline was found, make sure you\'ve got the correct '
                     'timeline ID or timeline name.'))
 
-        analyzer_name = form.analyzer_name.data
-        analyzer_kwargs = form.analyzer_kwargs.data
+        analyzer_name = form.get('analyzer_name')
```

Comment:
This is a list, so let's call it "analyzer_names"

Reply:
(offline discussions)